integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/AlterTableCommands.scala

```diff
@@ -89,9 +89,8 @@ private[sql] case class AlterTableAddColumns(
       LOGGER.info(s"Alter table for add columns is successful for table $dbName.$tableName")
       LOGGER.audit(s"Alter table for add columns is successful for table $dbName.$tableName")
     } catch {
-      case e: Exception =>
-        LOGGER.error(e, s"Alter table add columns failed : ${e.getMessage}")
-        // clean up the dictionary files in case of any failure
+      case runTimeException: RuntimeException => LOGGER
```

Comment:
why catch only `RuntimeException` ? why not `Exception` or any specific exception.